experimental/c-cpp/runner.py

```diff
@@ -251,6 +251,15 @@ def run_on_targets(target,
               build_heuristics=build_heuristics,
               generator_heuristics=generator_heuristics)
 
+  # Cleanup the OSS-Fuzz docker image
+  clean_up_cmd = [
+      'docker', 'image', 'rm', f'gcr.io/oss-fuzz/{worker_project_name}'
+  ]
+  try:
+    subprocess.check_call(' '.join(clean_up_cmd), shell=True)
```

Comment:
No need to join, IIRC?

Reply:
For `check_call` here it's needed -- a list doesn't do the job. That said, I'm not 100% sure about when subprocess wants a list/string in general.